packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -14,15 +14,18 @@ const SearchSuggestions = ({
   const getLinkToCourse = (course) => `/${enterpriseSlug}/course/${course.key}`;
   const getLinkToProgram = (program) => `/${enterpriseSlug}/program/${program.aggregation_key.split(':').pop()}`;
 
+  const courses = autoCompleteHits.filter(hit => hit.learning_type === 'course');
+  const programs = autoCompleteHits.filter(hit => hit.learning_type === 'program');
+  const execEdCourses = autoCompleteHits.filter(hit => hit.learning_type === 'Executive Education');
```

Comment:
[nit/curious] Kind of surprised `learning_type` comes back as a string like "Executive Education" vs. `executive_education_2u` or `executive-education-2u` or something like that?

Reply:
that was done intentionally to make the learning type a readable facet for the explore catalog